src/bundle.c

```diff
@@ -1019,13 +1021,227 @@ static int install_bundles(struct list *bundles, struct list **subs, struct mani
 	return ret;
 }
 
+static int lex_sort(const void *a, const void *b)
+{
+	const char *name1 = (char *)a;
+	const char *name2 = (char *)b;
+	return strcmp(name1, name2);
+}
+
+/* Get a list of files in a directory sorted by filename
+ * with their fullpath, returns NULL on error (errno set by
+ * opendir).
+ */
+static struct list *get_dir_files_sorted(char *path)
```

Comment:
I feel like these function belongs to some other file, like helpers.c, or since there are many functions related to this maybe its own alias.c file

Reply:
For the get_dir_files_sorted, putting it and lex_sort in helpers.c makes sense to me but because there isn't any other users outsdie of bundle.c I left it as is rather than polute the global namespace.

As for creating an alias.c, I think that would be fine but again as this only applies to bundle-add at the moment keeping them static seemed reasonable to me too.

It is not my project so just tell me whatever you all prefer and I'll do that =).

Reply:
ok, I'm ok leaving it as it is